File: sportify/models/game_level.go
package models

const (
	GameLevelLow      GameLevel = "low"
	GameLevelLowPlus  GameLevel = "low_plus"
	GameLevelMidMinus GameLevel = "mid_minus"
	GameLevelMid      GameLevel = "mid"
	GameLevelMidPlus  GameLevel = "mid_plus"
	GameLevelHigh     GameLevel = "high"
	GameLevelHighPlus GameLevel = "high+"
)

var enToRuGameLevel = map[GameLevel]string{ //nolint:gochecknoglobals
	GameLevelLow:      "начальный",
	GameLevelLowPlus:  "начальный плюс",
	GameLevelMidMinus: "средний минус",
	GameLevelMid:      "средний",
	GameLevelMidPlus:  "средний плюс",
	GameLevelHigh:     "полу-профи",
	GameLevelHighPlus: "профи",
}

func EnToRuGameLevel(level GameLevel) (string, bool) {
	result, ok := enToRuGameLevel[level]
	return result, ok
}

var ruToEnGameLevel = map[string]GameLevel{ //nolint:gochecknoglobals
	"начальный":      GameLevelLow,
	"начальный плюс": GameLevelLowPlus,
	"средний минус":  GameLevelMidMinus,
	"средний":        GameLevelMid,
	"средний плюс":   GameLevelMidPlus,
	"полу-профи":     GameLevelHigh,
	"профи":          GameLevelHighPlus,
}

func RuToEnGameLevel(level string) (GameLevel, bool) {
	result, ok := ruToEnGameLevel[level]
	return result, ok
}

type GameLevel string

File: bot/models/game_level.py
from enum import Enum


class GameLevel(Enum):
    LOW = "low"
    LOW_PLUS = "low_plus"
    MID_MINUS = "mid_minus"
    MID = "mid"
    MID_PLUS = "mid_plus"
    HIGH = "high"
    HIGH_PLUS = "high_plus"


en_to_ru_game_level = {
    GameLevel.LOW: "начальный",
    GameLevel.LOW_PLUS: "начальный+",
    GameLevel.MID_MINUS: "средний-",
    GameLevel.MID: "средний",
    GameLevel.MID_PLUS: "средний+",
    GameLevel.HIGH: "полу-профи",
    GameLevel.HIGH_PLUS: "профи",
}

def get_game_level_ru(sport_type: str) -> str:
    # TODO: check if present in map
    return en_to_ru_game_level[GameLevel(sport_type)]
